internal/pkg/levee/levee.go

```diff
@@ -35,6 +35,11 @@ var Analyzer = &analysis.Analyzer{
 	Requires: []*analysis.Analyzer{source.Analyzer, fieldpropagator.Analyzer},
 }
 
+type reportItems struct {
+	s *source.Source
+	v *ssa.Call
```

Comment:
Actually, within the context of "something we need to report," we should probably call `v -> c -> call -> sink`, since that is what we're reporting.  `s -> source` is a good \*ahem\* call, too.  Updated both.